### PR TITLE
Glitter-Finance V2 Bridge Update

### PR DESCRIPTION
**Pull request by Elaixzar**
**NOTE**

> - If you would like to add a `volume` adapter please submit the PR [here](https://github.com/DefiLlama/adapters).
> - If you would like to add a `liquidations` adapter, please refer to [this readme document](https://github.com/DefiLlama/DefiLlama-Adapters/tree/main/liquidations) for details.

1. Once your adapter has been merged, it takes time to show on the UI. If more than 24 hours have passed, please let us know in Discord.
2. Please enable "Allow edits by maintainers" while putting up the PR.
3. Sorry, We no longer accept fetch adapter for new projects, we prefer the tvl to computed from blockchain data, if you have trouble with creating a the adapter, please hop onto our discord, we are happy to assist you.
4. Please fill the form below  **only if the PR is for listing a new protocol** else it can be ignored/replaced with reason/details about the PR
5. **For updating listing info** It is a different repo, you can find your listing in this file: https://github.com/DefiLlama/defillama-server/blob/master/defi/src/protocols/data2.ts, you can  edit it there and put up a PR
6. Do not edit/push `package-lock.json` file as part of your changes, we use lockfileVersion 2, and most use v1 and using that messes up our CI
7. No need to go to our discord and announce that you've created a PR, we monitor all PRs and will review it asap

---
##### Github org/user (Optional, if your code is open source, we can track activity):

### CODE REVIEW

**Comment by llamatester** [comment]
The adapter at projects/glitter-finance exports TVL: 
        
 
 
    ethereum                  0
    arbitrum                  0
    polygon_zkevm             0
    avax                      0
    binance                   0
    polygon                   0
    optimism                  0
    algorand                  14.34 k
    solana                    32.74 k
    
    total                    47.07 k

**Comment by g1nt0ki** [comment]
hi @Elaixzar  I refactored and merged your code here: https://github.com/DefiLlama/DefiLlama-Adapters/pull/7113

File: projects/glitter-finance/index.js [hunk 1]
```diff
@@ -1,9 +1,42 @@
 const ADDRESSES = require('../helper/coreAssets.json')
-//Solana Helpers
 const { sumTokensExport, } = require("../helper/sumTokens")
+
 //addresses
 const SOLANA_VAULT = "7xCU4nvqu3Nz3BBQckKzibp3kBav4xbkuqQ3WM9CBHdJ";
 const ALGO_VAULT = "R7VCOR74LCUIFH5WKCCMZOS7ADLSDBQJ42YURFPDT3VGYTVNBNG7AIYTCQ";
+const ETHEREUM_VAULT = "0xa234acbd98a917f6dda69298e0e7290380006cf1";
```
Comment (g1nt0ki) [inline projects/glitter-finance/index.js:7]: what is the NOSEC token here?

on other evm chains, it is not a vault but bridged token?

Comment (Elaixzar) [reply projects/glitter-finance/index.js:7]: This is our vault for our bridged for glitter finance.  NOSEC gets bridged to xNOSEC on all the other chains.  THe vault on EThereum holds the bridged NOSEC.  We are in the process of listing the token with coingeko and cmc now.

**Comment by Elaixzar** [comment]
> hi @Elaixzar I refactored and merged your code here: #7113

This removed everything I added.  We added vaults for our new token we just released and are in the process of adding to CG and CMC.  

Can you please revert with the vaults included, even though the value currently shows 0, this will not always be the case once the token is added.

**Comment by g1nt0ki** [comment]
@Elaixzar just to be clear:

1. are those vaults on other chains or bridged version of NOSEC
2. is NOSEC deployed by your (glitter) team?

**Comment by Elaixzar** [comment]
>

**Comment by Elaixzar** [comment]
> @Elaixzar just to be clear:
> 
> 1. are those vaults on other chains or bridged version of NOSEC
> 2. is NOSEC deployed by your (glitter) team?

1.  They are vaults, but also bridged version.  How do we reflect we support these chains on our bridge, if we don't add these vaults?  Even if the balance is 0, we are ok with that so long as it reflects we support bridging to those chains.

2.  We support NOSEC on our bridge and hold the TVL in our vaults.

**Comment by g1nt0ki** [comment]
1. for bridged tokens, we count it only at source, right now, there is no way for us to show what chains a bridge support, but dont like the idea of listing bridges with 0 tvl.
2. so, totally unrelated teams?

**Comment by Elaixzar** [comment]
1.  That's fine.  We will work on adding more vaults on the other chains to show support then.
2.  There is some overlap, but different teams, yes.

**Pull request closed without merging**